source/structured-streaming.txt

```diff
@@ -37,6 +37,59 @@ Configuring a Write Stream to MongoDB
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         Specify write stream configuration settings on your streaming 
+         Dataset or DataFrame using the ``writeStream`` property. You 
+         must specify the following configuration settings to write 
+         to MongoDB:
+         
+         .. list-table::
+            :header-rows: 1
+            :stub-columns: 1
+            :widths: 10 40
+         
+            * - Setting
+              - Description
+         
+            * - ``writeStream.format()``
+              - The format to use for write stream data. Use 
+                ``mongodb``.
+         
+            * - ``writeStream.option()``
+              - Use the ``option`` method to specify your MongoDB 
+                deployment connection string with the 
+                ``spark.mongodb.connection.uri`` option key.
+         
+                You must specify a database and collection, either as 
+                part of your connection string or with additional 
```

Comment:
Suggestion: show an example of this using a connection URI

Reply:
I linked "connection string" to https://www.mongodb.com/docs/manual/reference/connection-string/ instead. I think most readers on this page will be familiar enough with connection strings already, and if not, the manual page should fill in the gaps.